File: spologp_v/lista2sala-C/questao3.c
#include<stdio.h>

int main()
{
    int A, B, diferenca;
    printf("Digite o valor de A:");
    scanf("%i", &A);
    printf("Digite o valor de B:");
    scanf("%i", &B);

    diferenca = A-B;

if(A>B){
    printf("A diferença do maior valor para o menor é: %i", diferenca);
}

else{
    diferenca = B-A;
    printf("A diferença do maior valor para o menor é: %i", diferenca);
}

return 0;
}

File: spologp_v/lista2sala-python/questao3.py
v1 = int(input("Digite o primeiro valor:"))
v2 = int(input("Digite o segundo valor:"))

diferenca = v1-v2

if(v1>v2):
    print("A diferença do maior para o menor valor é:", diferenca)
else:
    diferenca = v2-v1
    print("A diferença do maior valor para o menor é:", diferenca)